### ISSUE TITLE
Support NO_COLOR

### ISSUE THREAD

**Issue by SolraBizna** [SolraBizna/liso#4]
http://no-color.org/

**Comment by SolraBizna** [comment]
The `NO_COLOR` standard dictates that `NO_COLOR=1` make our program not *add* color by default. If I add support in the API that nulls out all color changes if the environment variable is present, then the application has no way to override this and produce color when explicitly instructed to. We would have to add a "really really add color, I mean it" feature. The way I see it, this is worse than not supporting `NO_COLOR` at all. So, for now, I'm closing this.

If you have strong opinions about `NO_COLOR` and wish to change my mind about this, feel free to comment on this issue.

tl;dr IMO `NO_COLOR` is for applications, not libraries.
